### ISSUE TITLE
widgets do not compose well with `cursor: default` 

### ISSUE THREAD

**Issue by maartenbreddels** [jupyter-widgets/ipywidgets#3692]
Due to lumino:
https://github.com/jupyterlab/lumino/blob/dbb4f6aae04996abfbc5ee17d5fa8e693f3eb024/packages/widgets/style/widget.css#L19

All our widgets gets `cursor: default`. This means that when an `Image` widget is a child of an anchor tag (e.g. it's embedded in a container that turns the image into a link), we don't get a "hand" icon.

Not sure if lumino shouldn't do this, or if we should reset this.

**Comment by jasongrout** [comment]
Yeah, I'm not sure that Lumino should do this either, it seems pretty heavy-handed if cursor is inherited. It looks like it was in since the beginning of Phosphor: https://github.com/jupyterlab/lumino/blame/42d1c2a646ac327da8b98f4a129a7d8947679ea6/styles/base.css#L25

**Comment by maartenbreddels** [comment]
Should we do fixes in both ipywidgets and in lumino? For 7.x we still use phosphor which might be difficult to fix right?

**Comment by jasongrout** [comment]
> For 7.x we still use phosphor which might be difficult to fix right?

Yes, it would be nigh impossible to fix Phosphor itself, but we could still do a CSS override in ipywidgets.

**Comment by maartenbreddels** [comment]
Fixed in 7.x (https://github.com/jupyter-widgets/ipywidgets/pull/3696), needs a forward port to main/8.x
